docs/static/_redirects

```diff
@@ -0,0 +1 @@
+/docs/development/setup /docs/development/setup/getting-started
```

Comment:
I think we wouldn't need this if we made the getting-started page `index.md`. This is how it's handled for Behaviors and Configuration Overview pages.